packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -742,6 +742,17 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
   }
 
   public async leaveCall(forEveryone?: boolean): Promise<void> {
+    const page = getCallCompositePage(
+      this.getState().call,
+      this.getState().endedCall,
+      this.getState().acceptedTransferCallState
+    );
+    if (page === 'transferring') {
```

Comment:
Isn't page available in this.getState().page?

Reply:
you are correct